src/en/models-config.md

```diff
@@ -103,12 +107,15 @@ type | string |  | any of the supported provider types | Type of model, e.g. loc
 uuid | string |  |  | The UUID of the model
 
 
-## Apt mirror
+## Examples
+
```

Comment:
Examples isn't really what is going on here anyhow. These are specific things which need further explanation and are often linked from the table. Perhaps, instead of having another heading, there should just be a paragraph of text that says something like:

   Many of these settings deserve further explanation. These topics are explored in more detail below.

or whatever